// @novu/shared/dist/cjs/types/organization.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.ProductUseCasesEnum = exports.ApiServiceLevelEnum = void 0;
exports.migrateServiceLevel = migrateServiceLevel;
var ApiServiceLevelEnum;
(function (ApiServiceLevelEnum) {
    ApiServiceLevelEnum["FREE"] = "free";
    ApiServiceLevelEnum["PRO"] = "pro";
    ApiServiceLevelEnum["BUSINESS"] = "business";
    ApiServiceLevelEnum["TEAM"] = "team";
    ApiServiceLevelEnum["ENTERPRISE"] = "enterprise";
    ApiServiceLevelEnum["UNLIMITED"] = "unlimited";
})(ApiServiceLevelEnum || (exports.ApiServiceLevelEnum = ApiServiceLevelEnum = {}));
function migrateServiceLevel(level) {
    switch (level) {
        case ApiServiceLevelEnum.UNLIMITED:
            return ApiServiceLevelEnum.ENTERPRISE;
        case ApiServiceLevelEnum.BUSINESS:
            return ApiServiceLevelEnum.TEAM;
        default:
            return level;
    }
}
var ProductUseCasesEnum;
(function (ProductUseCasesEnum) {
    ProductUseCasesEnum["IN_APP"] = "in_app";
    ProductUseCasesEnum["MULTI_CHANNEL"] = "multi_channel";
    ProductUseCasesEnum["DELAY"] = "delay";
    ProductUseCasesEnum["TRANSLATION"] = "translation";
    ProductUseCasesEnum["DIGEST"] = "digest";
})(ProductUseCasesEnum || (exports.ProductUseCasesEnum = ProductUseCasesEnum = {}));
